pathfinder/tests/test_graphs.py

```diff
@@ -1,28 +1,28 @@
 import random
+import time
 from math import isclose
 from typing import List
-import time
 
 import numpy as np
 import pytest
 from _pytest.monkeypatch import MonkeyPatch
 from networkx import NetworkXNoPath
-from raiden_libs.types import Address
 
 import pathfinder.model.token_network
-from pathfinder.model import ChannelView, TokenNetwork
 from pathfinder.config import DEFAULT_PERCENTAGE_FEE
+from pathfinder.model import ChannelView, TokenNetwork
+from raiden_libs.types import Address
 
 
 def test_routing_benchmark(
     token_network_model: TokenNetwork,
-    populate_token_network_random: None
+    populate_token_network_random: None,
 ):
     value = 100
```

Comment:
... and also add a comma here? Or did the linter already tell you where to put them?

Reply:
Why? this is just a statement

Reply:
(the linter tells you where to put the commas if you have the `flake8-commas` extension. And yes as paul said statements don't need commas. If you add a comma there you will get ... a tuple ;)

Reply:
good old linter, thanks!